### PR TITLE
Cleaner code

### PR DESCRIPTION
**Pull request by mbryzek**

### CODE REVIEW

File: src/main/scala/io/flow/proxy/Controller.scala [hunk 1]
```diff
@@ -23,7 +23,10 @@ case class Controller() extends io.flow.build.Controller {
   ) (
     implicit ec: scala.concurrent.ExecutionContext
   ) {
-    val services = allServices.filter { s => !ExcludeWhiteList.contains(s.name) && s.resources.length > 0 }
+    val services = allServices.
+      filter { s => !s.resources.isEmpty }.
```
Comment (chrhicks) [inline src/main/scala/io/flow/proxy/Controller.scala]: nitpick: `resources.nonEmpty`

**Comment by chrhicks** [comment]
lgtm

File: src/main/scala/io/flow/proxy/Controller.scala (hunk 1 above)
Comment (mbryzek) [inline src/main/scala/io/flow/proxy/Controller.scala]: done

Comment (ericluria) [inline src/main/scala/io/flow/proxy/Controller.scala]: We could also instead `s.resources.nonEmpty`, or `filterNot { s => s.resources.isEmpty }` for readability, but either way looks fine to me.

File: src/main/scala/io/flow/proxy/Controller.scala [hunk 2]
```diff
@@ -23,7 +23,10 @@ case class Controller() extends io.flow.build.Controller {
   ) (
     implicit ec: scala.concurrent.ExecutionContext
   ) {
-    val services = allServices.filter { s => !ExcludeWhiteList.contains(s.name) && s.resources.length > 0 }
+    val services = allServices.
+      filter { s => s.resources.nonEmpty }.
```
Comment (ericluria) [inline src/main/scala/io/flow/proxy/Controller.scala]: What about using `filterNot` to make the negation clearer?
```
filterNot { s => s.resources.isEmpty }.
filterNot { s => ExcludeWhiteList.contains(s.name) }
```

**Comment by mikeclimbrock** [comment]
lgtm

**Pull request merged**